package.json

```diff
@@ -24,6 +24,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "async": "2.0.0-rc.5",
```

Comment:
Actually, I can reproduce this locally if I create a super-basic **grunt-remarkable** example:

```
➜  remarkable-async grunt remarkable
Loading "remarkable.coffee" tasks...ERROR
>> Error: Cannot find module 'async'
Loading "remarkable.js" tasks...ERROR
>> Error: Cannot find module 'async'
Warning: Task "remarkable" not found. Use --force to continue.

Aborted due to warnings.
```

I can't reproduce this with a plain Grunt example, or something that uses **grunt-nsp** which makes me suspicious of **grunt-remarkable**. Digging deeper I'm super doubtful of this:

[**rwky/grunt-remarkable** tasks/remarkable.coffee:4](https://github.com/rwky/grunt-remarkable/blob/a833d6324ae1c83190ab5fcfd26f1e5038444d7d/tasks/remarkable.coffee#L4):

```coffee
async = require 'async'
```

Yet, I don't see **async** defined in the package.json's dependencies or devDependencies:
https://github.com/rwky/grunt-remarkable/blob/a833d6324ae1c83190ab5fcfd26f1e5038444d7d/package.json#L12-L17 (and I don't think there's a built-in Node.js **async** module):

```json
  "dependencies": {
    "remarkable": "1.6.x"
  },
  "devDependencies": {
    "coffee-script":"1.10.x"
  },
```

So adding async manually seems to be a hacky workaround, but fixes the issue. I'll file a bug w/ grunt-remarkable and see if we can get a fix pushed to npm.